lib/app/utils.js

```diff
@@ -132,11 +132,23 @@ export async function setContext(app, context) {
         path = status
         status = 302
       }
-      app.context.next({
-        path: path,
-        query: query,
-        status: status
-      })
+      if (path.charAt(0) === '/' && path.charAt(1) !== '/') {
```

Comment:
Does this means we don't support relative URLs anymore? Something like `./comments` will be an external URL :)

Reply:
@pi0 Yes, I just read the `vue-router` sources, I will make a change.